template.go

```diff
@@ -68,14 +68,6 @@ func generateTemplate(source, name string, delimLeft string, delimRight string)
 }
 
 func generateFile(templatePath, destinationPath string, debugTemplates bool, delimLeft string, delimRight string) error {
-	if !filepath.IsAbs(templatePath) {
```

Comment:
Can we make some tests covering this functionality?

Reply:
Sure thing, PTAL https://github.com/seznam/goenvtemplator/pull/25/commits/06358b5d42275b9cad65d393ebb36eea1e899b88